backend/README.md

```diff
@@ -0,0 +1,13 @@
+# Backend styles
+
+Http Status Codes
+--------------
+
+Status codes for common escenarious:
+
+* Social Login:
+	- `424 Failed Dependency`: If the user dont exist on the db.
```

Comment:
This 424, needs a better description. Something like if the social access token is for a valid user but the user doesn't exist on the db